proposed/2022/DotnetListPackageMachineReadableJsonOutput.md

```diff
@@ -772,6 +773,138 @@ Outputs json for format version 1, if it's not specified then latest version'll
 }
 ```
 
+### Error handling
+
+In case of an error it would be written into `problems` section.
```

Comment:
`npm -ls json` does same thing it creates `problems section` at top of json output.